operatorhub/openshift/config.yaml

```diff
@@ -220,16 +221,6 @@ image-substitutions:
 
 # add third party images which are not replaced by operator
 # but pulled directly by tasks here
-defaultRelatedImages:
-- image: registry.redhat.io/rhel8/buildah@sha256:82aa9592f3262313ec52f7a2335641e2581b0d0d9807980846d0539bb77d0657
-  name: IMAGE_ADDONS_PARAM_BUILDER_IMAGE
-- image: registry.redhat.io/openshift-serverless-1/client-kn-rhel8@sha256:1d5291d00c3f85de57b921b7450e41ceb58d8c3104c36135a242f5993531af5c
-  name: IMAGE_ADDONS_PARAM_KN_IMAGE
-- image: registry.redhat.io/rhel8/skopeo@sha256:3f16176d50e8132dd447aa5a513466a48d84e724bbb194d30501ae620a58172f
-  name: IMAGE_ADDONS_SKOPEO_COPY
-- image: registry.redhat.io/ocp-tools-4-tech-preview/source-to-image-rhel8@sha256:98d8cb3a255641ca6a1bce854e5e2460c20de9fb9b28e3cc67eb459f122873dd
-  name: IMAGE_ADDONS_GENERATE
-- image: registry.redhat.io/ubi8/ubi-minimal@sha256:3f32ebba0cbf3849a48372d4fc3a4ce70816f248d39eb50da7ea5f15c7f9d120
-  name: IMAGE_ADDONS_MVN_SETTINGS
```

Comment:
Why are we removing theses ?

Reply:
Now as the env are there in image substitution, we dont need here. Basically we adding twice then

Reply:
See here